IBX-8917: Content fixes in Developer Documentation (#2506)
ibexa/documentation-developer: docs/infrastructure_and_maintenance/security/reporting_issues.md

- If you're not a customer or partner, please log in to the [[= product_name_base =]] JIRA issue tracker: <https://issues.ibexa.co/>. Create an account if you don't have one, it's free. Click the "Create" button in the top menu to create your report. For "Project", select "[[= product_name_base =]] IBX", or "eZ Publish / Platform", or "eZ Platform Enterprise Edition", depending on which product is affected by the bug. **Important: Select "Security Level": "Security"!** The engineering team takes care of your report.
- It's also possible to report security issues by email to <security@ibexa.co> - this requires no account.

## Verbosity

Please be verbose when reporting issues. The issue is solved faster if you include:

- A **title** describing the gist of the issue in one sentence
- A **description** which includes the steps you take to produce the problem, what you expect the result to be, and what actually happens.

> - Make it clear **why you consider it a security issue**. If you know, also include what type of security issue it's (example: SQL injection, CSRF, Role/Policy failure), what the nature of it's (example: slowing/stopping a web site, leaking sensitive information, destroying data, privilege escalation) and how easy it's to exploit (example: Does it require editor login?).

Reviewer: It looks a bit weird to me, and still trigger the Microsoft.Contractions. Maybe with possessive determiners?

```suggestion
- Make it clear **why you consider it a security issue**. If you know, also include its type of security issue (example: SQL injection, CSRF, Role/Policy failure), its nature (example: slowing/stopping a web site, leaking sensitive information, destroying data, privilege escalation), and how easy it's to exploit (example: Does it require editor login?).
```
